components/raftstore-v2/src/operation/ready/async_writer.rs

```diff
@@ -124,11 +127,15 @@ impl<EK: KvEngine, ER: RaftEngine> AsyncWriter<EK, ER> {
         }
 
         let mut raft_messages = vec![];
+        let mut has_snapshot = false;
         // There must be a match in `self.unpersisted_readies`.
         loop {
             let Some(v) = self.unpersisted_readies.pop_front() else {
                 panic!("{:?} ready number not found {}", logger.list(), ready_number);
             };
+            if v.has_snapshot {
```

Comment:
You mean `hash_snapshot |= v.has_snapshot`?

Reply:
yes